src/cmd/services/m3coordinator/downsample/flush_handler.go

```diff
@@ -178,8 +179,8 @@ func (w *downsamplerFlushHandlerWriter) Write(
 				Value:     mp.Value,
 			}},
 			Unit: convert.UnitForM3DB(mp.StoragePolicy.Resolution().Precision),
-			Attributes: storage.Attributes{
-				MetricsType: storage.AggregatedMetricsType,
+			Attributes: consolidators.Attributes{
```

Comment:
Hm, can we keep these as "storage" attributes? Is it possible to put any types that depend on these types next to each other (if it was a circular dependency problem)?